generate_ideas.py

```
@@ -0,0 +1,209 @@
+
+# Load .env file
+env_path = Path(__file__).parent / ".env"
+if env_path.exists():
+    with open(env_path) as f:
+        for line in f:
+            line = line.strip()
+            if line and not line.startswith('#') and '=' in line:
+                key, value = line.split('=', 1)
+                os.environ[key] = value
+
+async def call_gemini_reliable(prompt: str, max_retries: int = 2) -> str:
```

Ruff (0.11.9)

`call_gemini_reliable` is too complex (12 > 10)

(C901)